kbase-extension/static/kbase/js/common/clock.js

```diff
@@ -1,43 +1,38 @@
-/*global define*/
-/*jslint white:true,browser:true*/
+define([], () => {
```

Comment:
No manual code changes; all diffs come from running `eslint --fix` and `prettier`.